usr/toolchain-nxdk.cmake

```diff
@@ -0,0 +1,46 @@
+set(NXDK_DIR $ENV{NXDK_DIR})
+
+set(CMAKE_SYSTEM_NAME Generic)
+set(CMAKE_SYSTEM_PROCESSOR Generic) #FIXME: Pentium 3
+set(TOOLCHAIN_PREFIX nxdk)
+
+set(WIN32 1)
+
+set(NXDK 1)
+
+# Keep depfiles to track header changes
+if(NOT "${lang}" STREQUAL "ASM")
```

Comment:
`lang` is a macro parameter - I don't use that style so far; I'll have to rewrite this to support this macro. See /usr/share/cmake-3.17/Modules/Platform/Windows-Clang.cmake and /usr/share/cmake-3.17/Modules/Platform/Windows-Clang-C.cmake and friends